src/analyzer.rs

```diff
@@ -634,7 +611,7 @@ mod test {
         // should have a member with a key to the map:
         let map = &root.members[0];
         assert_eq!(map.name, "validationsInfo");
-        assert_eq!(map.type_, "Option<BTreeMap<String, AgentValidationsInfo>>");
+        assert_eq!(map.type_, "Option<BTreeMap<String, Vec<AgentValidationsInfo>>>");
```

Comment:
double checked this to make sure, and [sure enough](https://www.redhat.com/en/blog/pxe-artifacts-in-red-hat-advanced-cluster-management-for-kubernetes), this was also wrong :+1: